pom.xml

```diff
@@ -1709,6 +1709,13 @@
                 <version>${dep.antlr.version}</version>
             </dependency>
 
+            <dependency>
+                <!-- Remove when Coral catches up with its transitive dependency version -->
+                <groupId>org.apache.ivy</groupId>
+                <artifactId>ivy</artifactId>
+                <version>2.5.1</version>
```

Comment:
What’s the issue with the version Coral depends on?

Reply:
If you guessed "security scanners" then you guessed right. The most likely outcome here is just that this is a false positive[^1], but I'm not very familiar with how this is used in Trino so I'm currently exploring possibilities here.

[^1]: It's used for one very specific thing which does not handle user-provided input.

Reply:
This has been pushed upstream and is no longer necessary.